project/sites/hiaredressni/themes/hiaredressni_theme/src/scss/components.scss

```diff
@@ -24,4 +24,39 @@
   }
 }
 
+#block-homepageabouttheredressboard {
+  display: flex;
+  text-align: left;
+  padding: 4rem 0 5rem;
+  border-bottom: 1px solid $brand-secondary;
+
+  .left {
+    flex: 2;
+  }
+
+  .right{
+    flex: 1;
+  }
+}
+
+.region-front-welcome .block h2 {
+  font-size: 4rem;
+  line-height: 4rem;
+  text-align: left;
+  font-weight: normal;
+}
+
+
+#main-menu .menu-link, #main-menu .menu {
+  background-color: #fff;
+  color: $brand-primary;
+}
+
+#main-menu .menu .menu-item .active, #main-menu .menu .menu-item .active + .menu-main_sub .title a {
```

Comment:
When targeting 2 selectors you should put the 2nd/3rd etc. on new lines for better readability.
You should also always use hex values or variables for colours. "blueviolet" should be a hex e.g. #878787

Reply:
Thanks Adam , I have now fixed this